app/locales/en/translations.js

```diff
@@ -146,7 +146,7 @@ const translations = {
         "title": "Your Personality"
     },
     "flag": {
-        "chooseLanguage": "Please select your country and language"
+        "chooseLanguage": "Please choose a language"
```

Comment:
It appears the above requested change will need to be made manually after all; for some reason github isn't being very smart about how it handles the merge of language changes.